workshops/forms.py

```diff
@@ -630,6 +644,12 @@ class SWCEventRequestForm(forms.ModelForm):
                   '-to-admin-fee.html" target="_blank">Look up administration '
                   'fees</a>.',
     )
+    language = selectable.AutoCompleteSelectField(
+        lookup_class=lookups.LanguageLookup,
+        label='Language',
+        required=False,
+        widget=selectable.AutoComboboxSelectWidget,
```

Comment:
`PersonCreateForm` inherits from `PersonForm` the langauge**s** field. 
The form at `workshops/persons/add` shows language**s** as a field.
Are you referring to the language field here in the `SWCEventRequest` form? I am not sure why I did not opt for a `ForeignKey` to the `Langauge` model on the model `EventRequest`. Do you want me to switch to M2M to `Language` model on the `EventRequest` model?